### DISCUSSION TITLE
Extension activation failed

### DISCUSSION THREAD

**Discussion by pdeberranger** [community/community#46627] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I've sign for a teacher account on github, it tells me ok for copilot but i've got :
Invalid copilot token: missing token: 403
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended

**Comment by SilasRodrigues19** [comment]
Hey @pdeberranger 

It seems that many people are facing this same problem.

I found 3 answered discussions on this topic, maybe it can help you:


#43634
#43695
#44054

**Reply by pdeberranger** [reply]
Hi, i don't think the answers in your cases are the real solution for me...

**Reply by SilasRodrigues19** [reply]
It was answered by one of the Maintainers that it is an authentication problem, try to redo this process to see if it fixes it

**Reply by SilasRodrigues19** [reply]
Check if it is active on your account too.

https://github.com/github-copilot/signup

**Comment by pdeberranger** [comment]
I think my account is good?
![2023-02-07 13_28_09-Clipboard](https://user-images.githubusercontent.com/58167894/217244894-6f634fd0-8b8e-4d49-ad0c-b5171e2b881d.png)

**Comment by pdeberranger** [comment, marked as answer]
The good solution is:
- logout from all accounts in visual studio code
- relog with only github
Copilot works !

**Reply by btvd** [reply]
I've tried, but it only works until restart of VS Code. Each time I restart it - each time Copilot fails to connect to server...
